responses/volume.go

```diff
@@ -0,0 +1,15 @@
+package responses
+
+import "dcfs/db/dbo"
+
+type VolumeDataSuccessResponse struct {
```

Comment:
hmm czy to nie jest identyczny obiekt taki, jak 
```
type EmptySuccessResponse struct {
	Success bool        `json:"success"`
	Data    interface{} `json:"data"`
}
```

Może niech poniższy konstruktor zwraca wypełnione EmptySuccessResponse? Tutaj data może (i powinno) być interface{}. Może nam to kiedyś się przydać, na wypadek, gdyby pojawił się import cycle

Reply:
tego response'a nie obchodzi jakiego typu są zwracane dane i nam to jest na rękę :D

Reply:
Słuszna uwaga, możemy uniknąć duplikowania responsów.